andela/ah-frontend-realers
# #166187810 Set up testing environment

#### What does this PR do?

Set up testing environment for project

#### Description of Task to be completed?

Install jest and enzyme
Create setupTest.js file
Set up testing environment

#### How should this be manually tested?

* -	Git fetch this repo locally in the terminal with `git fetch origin ch-tests-environment-166187810
* -	Install application dependencies: `npm install`
* -	Run application tests: `npm test`
* -	Check test coverage: `npx jest –coverage`

#### Any background context you want to provide?

This application did not have a testing environment nor tests, I have set up both.

#### What are the relevant pivotal tracker stories?

#[166187810](https://www.pivotaltracker.com/story/show/166187810)

#### Screenshots
Passing Tests

<img width="639" alt="Screenshot 2019-07-01 at 21 45 23" src="https://user-images.githubusercontent.com/32802973/60516013-7f60b980-9ce5-11e9-8c46-c5a987bb0ce7.png">

Tests with the coverage

<img width="813" alt="Screenshot 2019-07-01 at 21 50 37" src="https://user-images.githubusercontent.com/32802973/60516041-956e7a00-9ce5-11e9-97dd-d8d32764f560.png">
